basis/functors/functors.factor

```diff
@@ -159,16 +153,14 @@ DEFER: ;FUNCTOR delimiter
     [
         building get use-words
         (parse-bindings)
-        building get unuse-words
     ] with-bindings ;
 
 : parse-functor-body ( -- form )
-    push-functor-words
+    functor-words use-words
```

Comment:
I don't think so because it is only called from `parse-locals-definition` which sets up the "lambda scope" for it. I saw that you added a new scope for it and that should work too.